docs/data/base/components/select/select.md

```diff
@@ -0,0 +1,91 @@
+---
+product: base
+title: React Select unstyled component and hook
+components: SelectUnstyled, MultiSelectUnstyled, OptionUnstyled, OptionGroupUnstyled
+githubLabel: 'component: select'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#combobox
+---
+
+# Select
+
+<p class="description">Select components are used for choosing options from a list.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+MUI Base offers two components that aim to replace the native `<select>` tag: SelectUnstyled and MultiSelectUnstyled.
+
+## The SelectUnstyled component
+
+```tsx
+import SelectUnstyled from '@mui/base/SelectUnstyled';
+```
+
+### Basic usage
+
+{{"demo": "UnstyledSelectSimple.js"}}
+
+The `SelectUnstyled` is a component that accepts generic props.
+Due to Typescript limitations, this may cause unexpected behavior when wrapping the component in `forwardRef` (or other higher-order components).
+In such cases, the generic argument will be defaulted to `unknown` and type suggestions will be incomplete.
+To avoid this, manually cast the resulting component to the correct type (as shown above).
+
+The rest of the demos below will not use `forwardRef` for brevity.
+
+### Controlled select
+
+The SelectUnstyled can be used as either uncontrolled (as shown in the demo above) or controlled component.
```

Comment:
```suggestion
`SelectUnstyled` can be used as an uncontrolled (as shown in the demo above) or controlled component.
```

Reply:
We may want to explicitly state what this terminology means. I don't think I ever encountered it before this. Potential resource to draw from: [What are Controlled Components in React](https://www.robinwieruch.de/react-controlled-components/)

Reply:
This will likely occur frequently across Base docs, as most of the components can work in either controlled or uncontrolled mode. I don't think we should explain the concept on each page. We could write a few words about it on a page about general unstyled components anatomy instead. What do you think?

Reply:
Good content to be in a Getting Started environment of sorts!